app/models/claims/school.rb

```diff
@@ -78,4 +79,10 @@ class Claims::School < School
   def grant_conditions_accepted?
     claims_grant_conditions_accepted_at?
   end
+
+  def remove_grant_conditions_agreement
+    self.claims_grant_conditions_accepted_at = nil
+    self.claims_grant_conditions_accepted_by_id = nil
+    save!
+  end
```

Comment:
Let's avoid adding mutative methods to models. I'd suggest `#update!(claims_grant_conditions_accepted_at: nil, claims_grant_conditions_accepted_by: nil)` in the controller directly for this.

The alternative would be to add a service named `Claims::School::RemoveGrantConditionsAcceptance`.